experiment.py

```diff
@@ -179,6 +188,7 @@ def Scan_Field(stop):
     )
     client.set_temperature(
         300,
-        2,
+        3,
         client.temperature.approach_mode.fast_settle
     )
+    time.sleep(100 * 60)

```

Comment:
We should use client.wait_for here instead of this time.sleep. It should have the same effect.